Editorial work to refine auth implementation notes

zimeon opened the pull request:
I've taken a pass over the [auth implementation notes](http://iiif.io/api/auth/0.9/implementation/) and made a number of minor editorial changes. I think they are very nearly there. I have

  * Used either "IIIF Authentication Specification" or "specification", avoided intermediate "Authentication Specification".
  * Change to use US spelling: authorised -> authorized, behaviour -> behavior, customise -> customize, optimize -> optimize, summarise -> summarize
  * I have removed all use of "you" in the description, replacing with some variant of "the client" (IE section)
  * Fixed section numbering

Questions:
  * Should we link "XMLHttpRequest API" to https://xhr.spec.whatwg.org/ ?
  * I still find it awkward that we switch between countable and uncountable use of "script" -- anyone else?
  * Should there be some attempt to introduce the IE sections, including IE as an acronym? Seems inconsistent not to but I suspect everyone reading will understand it anyway.

zimeon commented:
This is more on #924

tomcrane commented:
Questions:

1) We currently have:
[xhr]: https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest "HttpXmlRequest"
[xdr]: https://developer.mozilla.org/en-US/docs/Web/API/XDomainRequest "XDomainRequest"
I guess whatwg doesn't have an entry for XDomainRequest though, so they'd be different. Does that matter?

2) - I didn't notice when I wrote it, not sure now:
 "the 302 status code will never be seen by client script" 
 "a malicious script that obtains an access token"

Still not sure! Any W3C spec guidance on this? I don't think I mind it.
http://www.oxfordlearnersdictionaries.com/definition/american_english/script_1 


3) It does seem inconsistent but I'm not sure how to introduce it without making it hard to read, unless we just change it in the headings and leave it in the body text.

azaroth42 commented on source/api/auth/0.9/implementation.md:
```diff
@@ -108,61 +110,61 @@ switch(myXhr.statusCode){
 To encourage the frequent use of requests to the token service to improve the user experience, a client should assume that the token service is a lightweight operation, and servers should ensure that they can handle frequent token requests. 
 
-## 6. Security for Server Implementors
+## 7. Security for server implementors
```
Why is this being lower cased?

zimeon replied:
We have sentence style for the other section headings

zimeon replied:
Oh, but I see we don't for the image API. Perhaps we should change all sections headings to Hollywood style?

azaroth42 commented:
:+1: overall

azaroth42 commented on source/api/auth/0.9/implementation.md:
```diff
@@ -108,61 +110,61 @@ switch(myXhr.statusCode){
 {: .note}
 
-## 7. Optimizations at the Presentation API level
+## 8. Optimizations at the Presentation API level
```
Presentation API is a proper noun, I guess? I prefer headings with caps, but can live with sentence case if others prefer it.

zimeon commented:
`/auth/0.9/` - Bumped version to Auth API to 0.9.4 with Wednesday's date, added log line. Title is now "IIIF Authentication API 0.9.4" per discussion on #926  (Apologies for mixing in the edits for auth spec with impl notes but it seems easiest to manage one branch that is "auth deploy for review").

`/auth/0.9/implementation/` - Refers to "IIIF Authentication API", now has link on "XMLHttpRequest API", no change to use of script and no attempt to use of 'IE' without intro. Fixed internal Jekyll page id but not sure if that affects anything.

mikeapp commented on source/api/auth/0.9/implementation.md:
```diff
@@ -57,46 +57,47 @@ At a minimum, a client implementation following a "happy path" would:
 
 There are different ways of doing implementing this in different styles of client user interface. These notes explore the issues implementors will encounter.
```
"doing implementing" -> "implementing"

mikeapp replied:
"implementors" -> "implementers"  http://dictionary.cambridge.org/us/dictionary/english/implementer (Also in TOC and section 7 header)

zimeon replied:
Thanks, both issues fixed in 2e74227

mikeapp commented:
👍

jpstroop commented:
I'm sorry I've been so out of touch--busy couple of weeks. I finally just managed to give the implementation notes a reasonably thorough readthrough and I think things are looking pretty good. I don't have any specific feedback other than that I think it's time we get the community to read it. Really nice job everyone (except me...)! 👍

azaroth42 commented:
:+1:s from 4 eds... merging.
